src/eval-frame/components/eval-container.jsx

```diff
@@ -36,6 +51,17 @@ export class EvalContainerUnconnected extends React.Component {
           <AppInfoPane />
         </FixedPositionContainer>
 
+      </React.Fragment>)
+    if (this.state.hasError) {
+      render = <ErrorPane error={this.state.error} errorInfo={this.state.errorInfo} />
+    }
+
+
+    return (
+      <React.Fragment>
+
+        {render}
+
```

Comment:
Some extra spacing/newlines in here that we don't need. We should probably configure the linter to flag these...

Reply:
ok, How much newlines space are allowed?

Reply:
When in doubt, look to the other jsx files for inspiration. In this case, I would do something like this (though note that we probably don't want the render element at all, see above):

```jsx
    if (this.state.hasError) {
      render = <ErrorPane error={this.state.error} errorInfo={this.state.errorInfo} />
    }
    return (
      <React.Fragment>
        {render} 
```